app/screens/home/search/search.tsx

```diff
@@ -2,29 +2,34 @@
 // See LICENSE.txt for license information.
 
 import {useIsFocused, useNavigation} from '@react-navigation/native';
-import {debounce} from 'lodash';
-import React, {useCallback, useState, useEffect} from 'react';
+import React, {useCallback, useMemo, useState} from 'react';
 import {useIntl} from 'react-intl';
 import {FlatList, StyleSheet} from 'react-native';
 import Animated, {useAnimatedStyle, withTiming} from 'react-native-reanimated';
 import {Edge, SafeAreaView} from 'react-native-safe-area-context';
 
+import {addSearchToTeamSearchHistory} from '@actions/local/team';
 import {searchPosts, searchFiles} from '@actions/remote/search';
 import FreezeScreen from '@components/freeze_screen';
+import Loading from '@components/loading';
 import NavigationHeader from '@components/navigation_header';
 import RoundedHeaderContext from '@components/rounded_header_context';
 import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
 import {useCollapsibleHeader} from '@hooks/header';
 import {FileFilter, FileFilters, filterFileExtensions} from '@utils/file';
 
 import Modifiers from './modifiers';
+import RecentSearches from './recent_searches';
 import Results from './results';
 import Header, {SelectTab} from './results/header';
 
 const EDGES: Edge[] = ['bottom', 'left', 'right'];
+const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
 
 const emptyFileResults: FileInfo[] = [];
 const emptyPostResults: string[] = [];
+const dummyData = [1];
```

Comment:
What is this dummy data for?

Reply:
This is a trick that Elias added when we rearranged the components in this file.  I will need to research this better before I have an answer.

Reply:
From what I can see, flatlist needs data to render, and this is providing that data (and does nothing with it). I still don't understand why the flat list, but we can get back to it later when Elias is back.